timesketch/api/v1/resources/sketch.py

```diff
@@ -235,6 +235,16 @@ def _remove_label(self, sketch, label):
         sketch.remove_label(label)
         return True
 
+    def _get_last_activity(self, sketch):
```

Comment:
Can you break this out to a utility function so we can reuse here as well as in the SketchListResource? Now we have this duplicated.

If not in this PR, then add a TODO :)

Reply:
I was just thinking about that, that we should rather do that, same with the updating of the timestamp, so that we can more easily re-use that in other resources, so let's do that now...